src/ui/server/controllers/insurance/business/company-details/index.POST.test.ts

```diff
@@ -42,6 +42,7 @@ describe('controllers/insurance/business/companies-details', () => {
           [COMPANY_HOUSE.INPUT]: req.body[COMPANY_HOUSE.INPUT],
           [TRADING_NAME]: sanitiseValue(req.body[TRADING_NAME]),
           [TRADING_ADDRESS]: sanitiseValue(req.body[TRADING_ADDRESS]),
+          [WEBSITE]: req.body[WEBSITE],
```

Comment:
Not for this PR - i'm wondering if there's anything we'd need to possibly sanitise for saving the data, what do you think?

Reply:
we should really sanitise it for code, but tech debt